lms/views/decorators/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+"""Python decorators meant to be used for decorating Pyramid view functions."""
+
+from __future__ import unicode_literals
+
+from lms.views.decorators.h_api import create_h_user
```

Comment:
I'm pretty happy with `lms.views.decorators` rather than `lms.view_decorators` as h has it.

It's a small thing but in a shell or in vim typing <kbd>lms/v<kbd>Tab</kbd></kbd> now autocompletes <samp>lms/views</samp> rather than annoyingly making you do more clicks to choose between completing <samp>views</samp> or <samp>view_decorators</samp> (a further <kbd>d<kbd>Tab</kbd></kbd> will then autocomplete to <samp>lms/views/decorators</samp>). Much nicer.

Also writing this PR has made me realise that view decorators are views and should be kept as thin as possible and have as much code moved out into services etc for the same reason as views should.

Reply:
> It's a small thing but in a shell or in vim typing lms/vTab now autocompletes lms/views rather than annoyingly making you do more clicks to choose between completing views or view_decorators (a further dTab will then autocomplete to lms/views/decorators). Much nicer.

Hear, hear!